Update wsl.rst (#1928)
pyansys/pymapdl: doc/source/getting_started/wsl.rst

Windows Subsystem for Linux
###########################

This page explains how you use a PyAnsys library, more specifically PyMAPDL,
in the Windows Subsystem for Linux (WSL). WSL is a compatibility layer for
running Linux binary executables natively on Windows 10, Windows 11, and
Windows Server 2019. For more information, see:

- `Wikipedia WSL <WikipediaWSL_>`_
- Microsoft's `What is the Windows Subsystem for Linux <What_is_the_Windows_Subsystem_for_Linux_>`_.

This page walk you through the installation of WSL on Windows and then
show how to use it together with MAPDL, PyMAPDL, and `Docker <https://www.docker.com/>`_.

.. caution::
   This approach hasn't been fully tested with a VPN connection. If you
   experience any problems connecting WSL to the internet, try to
   disconnect from the VPN.

Run PyMAPDL on WSL 
******************
There are two versions of WSL: WSL1 and WSL2. Because WSL2 provides many improvements
over WSL1, you should upgrade to and use WSL2.

Install WSL
===========

> 

Author: .. vale off

``/usr/ansys_inc``. Whether you install it there or not, you should
use a symbolic link to associate that directory with your Ansys installation
directory (``/*/ansys_inc``).

Post-installation setup
=======================

Open ports for license server communication
-------------------------------------------

**Theory:** You should open the ports ``1055`` and ``2325`` for license server
communication in the **Windows Control Panel**. For the steps to set advanced
Windows firewall options, see Microsoft's
`How to open port in Windows 10 Firewall <open_port_windows_10_>`_.

**Reality:** This works if you want to run a Docker image using WSL Linux image
to host that Docker image. The Docker image successfully communicates with the Windows
license server using these ports if you use the ``'-p'`` flag when running the
Docker image with these ports open. For more information, see `Run an MAPDL image <run_an_mapdl_image_>`.

If you want to run MAPDL in the CentOS 7 image and use the Windows license
server, opening the ports might not work properly because the Windows firewall
seems to block all traffic coming from WSL. For security purposes, you should

> still try to open ports ``1055`` and ``2325`` in the firewall and check if your

Author: ```suggestion
still try to open ports ``1055`` and ``2325`` in the firewall and verify that your
```
